Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule_test.py

```diff
@@ -400,6 +400,43 @@ def test_get_distribution_url(requests_mock):
     assert readable_output == f'[Distribution URL]({expected_url})'
 
 
+def test_download_distribution(requests_mock):
+    """
+    Given:
+        - Core client
+        - Distribution ID and package type
+    When
+        - Running xdr-download-distribution command
+    Then
+        - Verify filename
+        - Verify readable output is as expected
+    """
+    from CoreIRApiModule import download_distribution_command, CoreClient
+    
+    get_distribution_url_response = load_test_data('./test_data/get_distribution_url.json')
+    dummy_url = "https://xdrdummyurl.com/11111-distributions/11111/sh"
+    requests_mock.post(
+        f'{Core_URL}/public_api/v1/distributions/get_dist_url/', 
```

Comment:
please fix the pre-commit errors for this line.
In addition, please add unit tests and release notes